### PR TITLE
[Cordova] Add cordova 4.x support

### PR DESCRIPTION
**Pull request by zhuyongyong**
- Move some usecases to demo express
- Fail Analyse:XWALK-2233/2648

Impacted tests(approved): new 3, update 0, delete 0
Unit test platform: Crosswalk Project for android 18.46.457.0
Unit test result summary: pass 2, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-6056
BUG=https://crosswalk-project.org/jira/browse/XWALK-5864

### CODE REVIEW

File: README.md~ [hunk 1]
```diff
@@ -0,0 +1,151 @@
+## Introduction
```
Comment (Honry) [inline README.md~]: Please remove this temporary file.

File: VERSION~ [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Honry) [inline VERSION~]: Ditto.

File: samples-cordova/AdMob3.6/COPYING [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This test comes from
```
Comment (Honry) [inline samples-cordova/AdMob3.6/COPYING]: New cordova 3.6 tests are unnecessary in demo-express presently.

**Comment by zhuyongyong** [comment]
@Honry Pls review it

File: README.md [hunk 4]
```diff
@@ -40,20 +40,9 @@ Here is the buiding steps:
...
-    `$ mkdir -p /[userdir]/tools/crosswalk/`
-
-    `$ unzip crosswalk-<version\>.zip -d /[userdir]/tools/`
-
-    `$ cd /[userdir]/tools/`
-
-    `$ mv crosswalk-<version\>\* crosswalk/`
+    `$ ./tools/pack.py -t apk -m shared|embedded -a x86|arm --tools='/[userdir]/tools/'`
 
+* Pack cordova 3.6 webapi_cordova_demo Crosswalk Android binary,
```
Comment (Honry) [inline README.md]: Please remove cordova 3.6 related doc and tests.

**Comment by Honry** [comment]
LGTM

**Comment by Honry** [comment]
@jennycao, please also review it.

**Comment by jennycao** [comment]
LGTM

**Pull request merged**